dataactbroker/README.md

```diff
@@ -240,65 +240,74 @@ Example Output:
 ```
 
 #### POST "/v1/submit_files/"
-This route is used to retrieve S3 URLs to upload files. Data should be JSON with keys: ["appropriations", "award_financial", "award", "program_activity"], each with a filename as a value, and submission metadata keys: ["agency_name","reporting_period_start_date","reporting_period_end_date","is_quarter","existing_submission_id"].  If an existing submission ID is provided, all other keys are optional and any data provided will be used to correct information in the existing submission.
 
-This route will also add jobs to the job tracker DB and return conflict free S3 URLs for uploading. Each key put in the request comes back with an url_key containing the S3 URL and a key_id containing the job id. A returning submission_id will also exist which acts as identifier for the submission.
 
-A credentials object is also part of the returning request. This object provides temporarily access to upload S3 Files using an AWS SDK. It contains the following: SecretAccessKey, SessionToken, Expiration, and AccessKeyId.
-It is important to note that the role used to create the credentials should be limited to just S3 access.
+A call to this route should be of content type `"multipart/form-data"`, and should use @ notation for the values of the "appropriations", "program_activity" and "award_financial" keys, to indicate the local path to the files to be uploaded.
+If an existing submission ID is provided, all other keys are optional and any data provided will be used to correct information in the existing submission.
 
-When upload is complete, the finalize_submission route should be called with the job_id.
+This route will upload the files, then kick off the validation jobs. It will return the submission id, which can be used for the `/v1/check_status/` route to poll for validation completion. 
 
-Example input:
 
-```json
-{
-  "appropriations":"appropriations.csv",
-  "award_financial":"award_financial.csv",
-  "award":"award.csv",
-  "program_activity":"program_activity.csv",
-  "agency_name":"Name of the agency",
-  "reporting_period_start_date":"03/31/2016",
-  "reporting_period_end_date":"03/31/2016",
-  "is_quarter":False,
-  "existing_submission_id: 7 (leave out if not correcting an existing submission)
-}
-```
-
-Example output:
-
-```json
-{
-  "submission_id": 12345,
+Required Headers:
+        `X-Session-ID`: string, session token id
```

Comment:
`X-Session-ID` is always a required header, I don't know if we need to include it here. We may want to rename this to `Additional Required Headers:` and just include the `Content-Type`.